src/categories/clothing.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Clothing'
+image: ../images/clothing.jpg
+intro: The best brands are ones that use ethical production and sustainable materials
+---
+
+Sustainable clothing comes in many shapes and forms. One example is through ethical manufacturing which respects both the environment as well as humans living in surrounding areas - such as avoiding the use of harmful dyes. Another form can come from clothing made with natural materials such as:
```

Comment:
If you dont mind editorial notes in general here, yes we do want people to buy first hand clothing made out of good things but only as a last resort. Buying less clothing (by focusing on buying durable clothing and repairing instead of replacing) is a important factor, and buying second hand is even more important. 

Instead of buying cheap shit from H&M and replacing it every few months when it breaks, I buy some slightly more expensive stuff first hand, and some cheap but still awesome stuff second hand. I then make this last forever by sowing up rips.

I bought some old amazing second hand leather boots for €7 then resoled them for €20 and they beat the hell out of any modern boots I could have got for €100. This is the kind of thinking we want to express, telling people they can avoid fast fashion, have amazing things, and not pay loads of money. Sustainability is not for the well off, its for everyone, we just need to shake the stigma. If I have a sown up tear in my clothes its not because im "poor", its because I dont want to fill up landfills with something I could easily fix.

Mentioning tencel could be good too. :D

Reply:
Will update this section. Can't believe I forgot about buying less and second-hand articles...